### PR TITLE
rollback nock to 11.0.0-beta.14

### PR DESCRIPTION
**Pull request by calebcartwright**
Refs https://github.com/badges/shields/pull/3552#issuecomment-504615180

Revert version bump of nock to fix service test issues that appear with nock 11.0.0-beta.19

### CODE REVIEW

**Comment by calebcartwright** [comment]
~7 service test failures, and those were just some of the typical timeout etc. types of errors. 

Reverting the nock version has the desired result, so removing the service test targets from the title

**Comment by calebcartwright** [comment]
Error deploying review app related to anafanafo module not found. Not sure what's happening here

```sh
-----> Build
       Detected both "build" and "heroku-postbuild" scripts
       Running heroku-postbuild
       
       > shields.io@0.0.0 heroku-postbuild /tmp/build_57033e2f3fad2bfd9c242d6554780ab6
       > run-s --silent build
       
internal/modules/cjs/loader.js:626
    throw err;
    ^
Error: Cannot find module 'anafanafo'
Require stack:
- /tmp/build_57033e2f3fad2bfd9c242d6554780ab6/gh-badges/lib/make-badge.js
- /tmp/build_57033e2f3fad2bfd9c242d6554780ab6/core/base-service/base-non-memory-caching.js
- /tmp/build_57033e2f3fad2bfd9c242d6554780ab6/core/base-service/index.js
- /tmp/build_57033e2f3fad2bfd9c242d6554780ab6/services/index.js
- /tmp/build_57033e2f3fad2bfd9c242d6554780ab6/services/amo/amo-downloads.service.js
- /tmp/build_57033e2f3fad2bfd9c242d6554780ab6/core/base-service/loader.js
- /tmp/build_57033e2f3fad2bfd9c242d6554780ab6/scripts/export-service-definitions-cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
```

**Comment by paulmelnikow** [comment]
Hrm, did the latest master deploy to Heroku okay? If so, I wonder if the lockfile updates are causing this.

**Comment by calebcartwright** [comment]
> Hrm, did the latest master deploy to Heroku okay

Yeah looks like master deployed just fine

File: package-lock.json [hunk 1]
```diff
@@ -12222,18 +12225,21 @@
         },
         "anafanafo": {
           "version": "1.0.0",
-          "bundled": true,
+          "resolved": "https://registry.npmjs.org/anafanafo/-/anafanafo-1.0.0.tgz",
+          "integrity": "sha512-pDPbI7SFRHu0byMXHAf+v74+LCcHSxnLYkcbfiV91XRlE+NSLpFCpEQdVUy9ZxZw/LuhTrOin4r8wlR3OFrKBA==",
```
Comment (paulmelnikow) [inline package-lock.json]: Interesting, when I ran `npm install` it did change something related to anafanafo. I'm on npm 6.9.0, how about you?

Comment (calebcartwright) [reply package-lock.json]: 6.2.0

Comment (paulmelnikow) [reply package-lock.json]: Hmm, you might want to upgrade. It looks like this is deploying fine.

**Pull request merged**